defaults/main.yml

```diff
@@ -11,7 +11,7 @@ haproxy_extra_packages: "{{ _haproxy_extra_packages }}"
 
 haproxy_manage_config: true
 
-_haproxy_ssl_options: 'no-sslv3 no-tls-tickets'
+_haproxy_ssl_options: 'no-sslv3 no-tls-tickets ssl-min-ver TLSv1.2'
```

Comment:
to keep compatibility with haproxy prior to 1.8
`no-sslv3 no-tlsv10 no-tlsv11 no-tls-tickets`

note that `no-sslv3` is useless with `ssl-min-ver TLSv1.2`

1.7 EOL is 2021-Q4

Reply:
Good point. @twillouer can you update the PR with @aodren's suggestion?